hollow/src/main/java/com/netflix/hollow/api/metrics/HollowConsumerMetrics.java

```diff
@@ -18,9 +18,31 @@
 
 import com.netflix.hollow.core.read.engine.HollowReadStateEngine;
 
+import java.util.concurrent.atomic.AtomicLong;
+
 public class HollowConsumerMetrics extends HollowMetrics {
     private int refreshFailed;      // TODO: Move these metrics over to com.netflix.hollow.api.consumer.metrics.AbstractRefreshMetricsListener
     private int refreshSucceeded;
+    private AtomicLong lastRefreshStartTimeNs; //last time a refresh started
+    private AtomicLong lastRefreshEndTimeNs;
+
+    public void setLastRefreshStartTimeNs() {
+        lastRefreshStartTimeNs = new AtomicLong(System.nanoTime());
```

Comment:
I think we're going for thread safety here.. Assigning a non volatile variable to a new AtomicLong wont achieve that

Reply:
fixed it and updated the PR